imports/plugins/core/router/client/app.js

```diff
@@ -36,6 +36,35 @@ const styles = {
   }
 };
 
+
+const handleViewContextChange = (event, value) => {
+  Reaction.setUserPreferences("reaction-dashboard", "viewAs", value);
+
+  if (Reaction.isPreview() === true) {
+    // Save last action view state
+    const saveActionViewState = Reaction.getActionView();
+    Reaction.setUserPreferences("reaction-dashboard", "savedActionViewState", saveActionViewState);
+
+    // hideActionView during isPreview === true
+    Reaction.hideActionView();
+  } else {
+    // // Reload previous actionView, if saved. Otherwise, don't show.
```

Comment:
If we aren't using this code any more let's just remove it? Sorry I didn't catch that earlier.

Reply:
Cool, removing the else portion. Didn't remove it before because I thought it might be something we have disabled temporarily.